pkgs/applications/video/kodi-packages/inputstream-ffmpegdirect/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, rel, buildKodiBinaryAddon, fetchFromGitHub, kodi, bzip2, zlib }:
+
+buildKodiBinaryAddon rec {
+  pname = "inputstream-ffmpegdirect";
+  namespace = "inputstream.ffmpegdirect";
+  version = "1.19.4";
+
+  src = fetchFromGitHub {
+    owner = "xbmc";
+    repo = "inputstream.ffmpegdirect";
+    rev = "${version}-${rel}";
+    sha256 = "1ppvs6zybbi73zq1qh8klyhj99byh61c6nijmb1gd5yhg7cywf72";
+  };
+
+  extraBuildInputs = [ bzip2 zlib kodi.ffmpeg ];
```

Comment:
This is that all packages use the same ffmpeg?

Reply:
I am sorry but I do not understand your question. Can you please elaborate?

Reply:
Why do we use kodi.ffmpeg instead of ffmpeg?

Reply:
`kodi` has its own fork of `ffmpeg` which diverges from upstream in significant ways.